### DISCUSSION TITLE
Add options for output formatting

### DISCUSSION THREAD

**Discussion by wAuner** [jotaen/klog#383] [Something else]
I'm tinkering around with the alfred integration. It would be helpful to have options for create, start, stop etc to get unformatted plain text or even json returned, instead of the colorized text output. Maybe something like `klog stop --raw`

**Comment by jotaen** [comment]
Cool to hear! Uncoloured output shouldn’t be a problem (i.e. no ANSI sequences in between), and it’s probably useful in itself anyways. Could e.g. be one of:

- `--raw`
- `--plain`
- `--no-color`
- [`--color=no`](https://www.gnu.org/software/gettext/libtextstyle/manual/html_node/The-_002d_002dcolor-option.html#The-_002d_002dcolor-option)
- [`NO_COLOR`](https://www.gnu.org/software/gettext/libtextstyle/manual/html_node/The-NO_005fCOLOR-variable.html)

Providing JSON for the regular commands is more tricky unfortunately, because the implementation of the “CLI frontend” is currently tailored to plain-text output. (Except the `json` subcommand.)

**Comment by wAuner** [comment]
ok plain text would probably be enough for my purposes. All I want is to give visual feedback when a new entry is created or start/stop triggered.
